### PR TITLE
Create ATTACK.md

### PR DESCRIPTION
**Pull request by blockcreators**
First draft of attack on testnet

### CODE REVIEW

**Comment by JakeHartnell** [comment]
Would be nice to have a simple table with a leaderboard.

File: uni/ATTACK.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Junø - Attack On Testnet
+
+![att2](https://user-images.githubusercontent.com/63689287/141705292-86e3daa5-5616-4cda-a03e-ffeb1e445857.png)
+
+_Planned Start Time: December 1st at 1 PM UTC._
```
Comment (JakeHartnell) [inline uni/ATTACK.md:5]:
```suggestion
```

**Comment by blockcreators** [comment]
> Would be nice to have a simple table with a leaderboard.

Added table

File: uni/ATTACK.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+#1 | empty | 0
+#2 | empty | 0
+#3 | empty | 0
+#4 | empty | 0
+#5 | empty | 0
+#6 | empty | 0
+#7 | empty | 0
+#8 | empty | 0
+#9 | empty | 0
+#10 | empty | 0
```
Comment (JakeHartnell) [inline uni/ATTACK.md:21]:
```suggestion
Please report vulnerabilities to `Meow Stargaze ✨🔭#1736` or `dimi 🦙#2998` on Discord. Rewards will be determined based on the severity.
```

Comment (JakeHartnell) [reply uni/ATTACK.md:21]: > > Would be nice to have a simple table with a leaderboard.
> 
> Added table and also link on the uni readme.

Changed my mind on the table if it's all empty. Just suggested a change to suggest they reach out to a technical core team member.

**Review by JakeHartnell — APPROVED** [review]

**Pull request merged**